src/ceph-disk/tests/ceph-disk.sh

```diff
@@ -59,16 +64,51 @@ function teardown() {
         return
     fi
     kill_daemons $dir
-    if [ $(stat -f -c '%T' .) == "btrfs" ]; then
+    if [ `uname` != FreeBSD ] && \
+       [ $(stat -f -c '%T' .) == "btrfs" ]; then
         rm -fr $dir/*/*db
         __teardown_btrfs $dir
     fi
-    grep " $(pwd)/$dir/" < /proc/mounts | while read mounted rest ; do
+    grep " $(pwd)/$dir/" < ${PROCDIR}/proc/mounts | while read mounted rest ; do
         umount $mounted
     done
     rm -fr $dir
 }
 
+function timeout_diagnose() {
```

Comment:
this function helps if we are debugging the test, but it should be not used in the test, i think. if the test itself is sane, `timeout` here should not return 125,126,127 because we are passing the wrong argument to it, or the launched executable does not exists. the only possibility is that the "command" returned this status code.